src/components/Navbar.js

```diff
@@ -0,0 +1,18 @@
+import React, { Component } from "react";
+import { Link } from "react-router-dom";
+
+class NavBar extends Component {
+  render() {
+    const Separator = () => <span> &middot; </span>;
+    return (
+      <div>
+        <Link to="/">Home</Link>
+        <Separator />
+        <Link to="/kanban">Kanban View</Link>
+        <Separator />
+        <Link to="/list">List View</Link>
+      </div>
```

Comment:
Maybe you can get benefit of the `<ul><li>` here instead of creating **Separator**.

Reply:
I prefer the look of te Separator but it's the same. I changed it.

Reply:
I said it because I think it is easier for adding CSS in the future. In this time we don't get in the look of project, we are working on the functionality. Later we can think about the look of components.